src/frontend/components/DashboardVideoLive/index.spec.tsx

```diff
@@ -15,6 +21,13 @@ jest.mock('jwt-decode', () => jest.fn());
 jest.mock('../../data/appData', () => ({
   appData: { jwt: 'cool_token_m8' },
 }));
+jest.mock('../DashboardVideoLiveRaw', () => (props: { video: Video }) => {
+  return <span title={props.video.id} />;
+});
```

Comment:
Just to homegenize with the mock below and because it's prettier.
```suggestion
jest.mock('../DashboardVideoLiveRaw', () => (props: { video: Video }) => (
  <span title={props.video.id} />
));
```